content/en/synthetics/api_tests/_index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: API Tests
+kind: documentation
+beta: true
+description: Simulate and monitor HTTP requests from specific locations
+aliases:
+  - /synthetics/uptime_check
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetics"
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/browser_tests"
+  tag: "Documentation"
+  text: "Configure a Browser Test"
+---
+
+## Overview
+
+API tests are useful to help you monitor your API endpoints and alert you when they are failing or too slow.
+They are executed periodically from several locations around the world.
+
+Available locations are:
+
+- _Canada Central (AWS_)
+- _Ohio (AWS_)
+- _Oregon (AWS_)
+- _Sydney (AWS_)
+- _Tokyo (AWS_)
+- _Frankfurt (AWS_)
+- _London (AWS_)
```

Comment:
The formatting here is a little weird, is this intentional?

Reply:
nice shot, it is not!